src/Microsoft.AspNetCore.SignalR.Core/IGroupManager.cs

```diff
@@ -7,7 +7,7 @@ namespace Microsoft.AspNetCore.SignalR
 {
     public interface IGroupManager
     {
-        Task AddAsync(string connectionId, string groupName);
-        Task RemoveAsync(string connectionId, string groupName);
+        Task AddAsync(string groupName, string connectionId);
```

Comment:
I gotta say this change I’m not sure about m. It’s going to break all of the apps and I’m not even sure it’s right... maybe we should instead rename it AddToGroup